### PR TITLE
Updating to v1.3.0

### PR DESCRIPTION
**Pull request by delucchi-cmu**
All automated changes.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -21,16 +22,15 @@ dependencies = [
 # On a mac, install optional dependencies with `pip install '.[dev]'` (include the single quotes)
 [project.optional-dependencies]
 dev = [
+    "mypy", # Used for static type checking of files
+  "pre-commit", # Used to run checks before finalizing a git commit
```
Comment (OliviaLynn) [inline pyproject.toml:26]:
```suggestion
    "pre-commit", # Used to run checks before finalizing a git commit
```

Comment (OliviaLynn) [reply pyproject.toml:26]: Another indentation thing--I'll go in to the template to check these over afterwards

File: tests/lsdb/conftest.py
Comment (OliviaLynn) [inline tests/lsdb/conftest.py:1]: Is this to provide some kind of coverage for Sphinx's conf.py/is it supposed to be blank for now?

Comment (delucchi-cmu) [reply tests/lsdb/conftest.py:1]: This gets generated by the template, but is a pytest thing and basically tells pytest "here be unit tests"

File: docs/Makefile
[the remark this answers is not in the document]
Comment (delucchi-cmu) [reply docs/Makefile:1]: We probably want notebooks in here eventually, so there's no harm here. It might make sense to drop from the template, though.

**Pull request merged**